### PR TITLE
Bug 1889954: bump(*) to mitigate CVE-2020-8564

### PR DESCRIPTION
**Pull request by coreydaley**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -37,7 +37,7 @@ require (
 	github.com/moby/buildkit v0.6.2 // indirect
 	github.com/opencontainers/runc v1.0.0-rc9
 	github.com/opencontainers/runtime-spec v1.0.0
-	github.com/openshift/api v3.9.1-0.20191024142031-e89ff1fddcbe+incompatible
+	github.com/openshift/api v0.0.0-20200618202633-7192180f496a
```
Comment (adambkaplan) [inline go.mod:40]: Where did this api bump come from? This appears to be from a later OpenShift release.

**Review by adambkaplan — CHANGES_REQUESTED** [review]

File: go.mod (hunk 1 above)
Comment (coreydaley) [reply go.mod:40]: That is the latest commit in the release-4.4 branch of the openshift/api repository: https://github.com/openshift/api/commit/7192180f496aab1f7659d8660fc360498bab498b

Comment (adambkaplan) [reply go.mod:40]: Please indicate in the commit message that you are bumping openshift/api, and the reason for doing so. I assume you did this to get around an "invalid pseudo-version" error? I found something similar in a separate item I was iterating on.

File: go.mod [hunk 2]
```diff
@@ -54,8 +54,8 @@ require (
 	go.uber.org/zap v1.11.0 // indirect
 	golang.org/x/sys v0.0.0-20191127021746-63cb32ae39b2
 	golang.org/x/time v0.0.0-20190921001708-c4c64cad1fd0 // indirect
-	k8s.io/api v0.17.0
-	k8s.io/apimachinery v0.17.0
+	k8s.io/api v0.17.13
```
Comment (adambkaplan) [inline go.mod:57]: Please indicate in the commit message that you are bumping k8s to address this CVE.

**Review by adambkaplan — CHANGES_REQUESTED** [review]
Please update the git commit message and PR description to provide more details.

File: go.mod (hunk 2 above)
Comment (coreydaley) [reply go.mod:57]: updated

**Comment by coreydaley** [comment]
Commit messages updated

**Review by adambkaplan — APPROVED** [review]
/lgtm

**Comment by wewang58** [comment]
I see, The repo does not support qe-approve label

**Comment by russellb** [comment]
(patch manager) If this is CVE related, the severity should probably be bumped?  We typically only merge high/urgent in 4.4 right now

**Comment by crawford** [comment]
These failures seem to be pretty common. @coreydaley can you look into this?

```
 [image_ecosystem][Slow] openshift images should be SCL enabled  returning s2i usage when running the image [Top Level] [image_ecosystem][Slow] openshift images should be SCL enabled  returning s2i usage when running the image "registry.redhat.io/rhscl/php-73-rhel7" should print the usage [Suite:openshift]
[image_ecosystem][Slow] openshift images should be SCL enabled  returning s2i usage when running the image [Top Level] [image_ecosystem][Slow] openshift images should be SCL enabled  returning s2i usage when running the image "registry.redhat.io/rhscl/python-27-rhel7" should print the usage [Suite:openshift] 
```

/hold

**Comment by gabemontero** [comment]
> These failures seem to be pretty common. @coreydaley can you look into this?
> 
> ```
> [image_ecosystem][Slow] openshift images should be SCL enabled  returning s2i usage when running the image [Top Level] [image_ecosystem][Slow] openshift images should be SCL enabled  returning s2i usage when running the image "registry.redhat.io/rhscl/php-73-rhel7" should print the usage [Suite:openshift]
> [image_ecosystem][Slow] openshift images should be SCL enabled  returning s2i usage when running the image [Top Level] [image_ecosystem][Slow] openshift images should be SCL enabled  returning s2i usage when running the image "registry.redhat.io/rhscl/python-27-rhel7" should print the usage [Suite:openshift] 
> ```
> 
> /hold

ah ... if this PR is hitting that @coreydaley @crawford this was address in master/4.7 via an fix to the test in openshift/origin via https://github.com/openshift/origin/pull/25799

to date it has only been backported to 4.6 via https://github.com/openshift/origin/pull/25804

if we want to hold this on that test, vs. overriding it, we can add a 4.5 cherrypick  to the 4.6 PR, get it merged, then do  the 4.4 from the merged 4.5 PR

which way do you want to proceed @crawford ?

**Comment by crawford** [comment]
@gabemontero let's backport the test. Can you trigger those for me?

**Comment by gabemontero** [comment]
> @gabemontero let's backport the test. Can you trigger those for me?

https://github.com/openshift/origin/pull/25816 up for 4.5 @crawford ... added assignments for requisite approve/lgtm and cherrypick for 4.4 in that PR

**Comment by crawford** [comment]
https://github.com/openshift/origin/pull/25817 has merged.

/hold cancel

**Pull request merged**